### PR TITLE
Andranik sargsyan java classes homework 4

### PR DESCRIPTION
**Pull request by ArmSargsyan**

### CODE REVIEW

File: src/main/java/javaClasses/homework_4/AndranikSargsyan/QualityEngineer.java [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    private String fieldType;
+
+    public QualityEngineer(String field){
+        this.fieldType = field;
+    }
+
+    @Override
+    public String doWork() {
+        setName("Jon");
+        setCompany("EGS");
```
Comment (vkotsinyan11) [inline src/main/java/javaClasses/homework_4/AndranikSargsyan/QualityEngineer.java:14]: Why you set name and company inside doWork? Please change, it just only need to print info, without setting it

Comment (vkotsinyan11) [inline src/main/java/javaClasses/homework_4/AndranikSargsyan/QualityEngineer.java:14]: Why you set name and company inside doWork? Please change, it just only need to print info, without setting it

File: src/main/java/javaClasses/homework_4/AndranikSargsyan/Developer.java [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+public class Developer extends Staff {
+
+    private final String techStack;
+
+    public Developer(String skill, String name, String surName, String company) {
+        this.techStack = skill;
+        super.setName(name);
+        super.setSurName(surName);
+        Staff.setCompany(company);
```
Comment (vkotsinyan11) [inline src/main/java/javaClasses/homework_4/AndranikSargsyan/Developer.java:11]: Very nice!!! Good job!

Comment (vkotsinyan11) [inline src/main/java/javaClasses/homework_4/AndranikSargsyan/Developer.java:11]: Very nice!!! Good job!

**Review by vkotsinyan11 — CHANGES_REQUESTED** [review]

File: src/main/java/javaClasses/homework_4/AndranikSargsyan/QualityEngineer.java (hunk 1 above)
Comment (ArmSargsyan) [reply src/main/java/javaClasses/homework_4/AndranikSargsyan/QualityEngineer.java]: Done

File: src/main/java/javaClasses/homework_4/AndranikSargsyan/Calculations.java [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+        int result = 0;
+        for (int n : nums)
+            result += n;
+        return result / nums.length;
+    }
+
+    public double getMean(double... nums) {
+        int result = 0;
+        for (double n : nums)
+            result += n;
```
Comment (vkotsinyan11) [inline src/main/java/javaClasses/homework_4/AndranikSargsyan/Calculations.java:15]: In this case you might lose some data. For example your n is 2.5 if you assign it to int result you'll store only 2. Please review that part, and on the other code parts below

File: src/main/java/javaClasses/homework_4/AndranikSargsyan/QualityEngineer.java [hunk 4]
```diff
@@ -0,0 +1,16 @@
+package javaClasses.homework_4.AndranikSargsyan;
+
+public class QualityEngineer extends Staff{
+
+    private String fieldType;
+
+    public QualityEngineer(String field, String name){
+        this.fieldType = field;
+        super.setName(name);
```
Comment (vkotsinyan11) [inline src/main/java/javaClasses/homework_4/AndranikSargsyan/QualityEngineer.java:9]: I think here should be also last name

**Review by vkotsinyan11 — CHANGES_REQUESTED** [review]

File: src/main/java/javaClasses/homework_4/AndranikSargsyan/Calculations.java (hunk 3 above)
Comment (ArmSargsyan) [reply src/main/java/javaClasses/homework_4/AndranikSargsyan/Calculations.java:15]: done

File: src/main/java/javaClasses/homework_4/AndranikSargsyan/QualityEngineer.java (hunk 4 above)
Comment (ArmSargsyan) [reply src/main/java/javaClasses/homework_4/AndranikSargsyan/QualityEngineer.java:9]: done

**Review by vkotsinyan11 — APPROVED** [review]

**Review by vkotsinyan11 — APPROVED** [review]

**Pull request merged**